app/views/posts/_form_template.html.erb

```diff
@@ -0,0 +1,104 @@
+<div>&nbsp</div>
```

Comment:
There are quite a few things to change about this file and I think you know this, based on your email. The biggest thing is that the categories are hard coded. Let's change this to use a loop through all the categories in @all_needs first, and I'll get to the little details later. If Dan is asking for the other changes in this PR ASAP, then just remove this feature from the PR by checking out an old commit. We'll merge the other stuff first and revisit this show/hide feature.

Reply:
And I understand you hardcoded this because of the Javascript. There's a way to do this by using the .parent() and .child() jQuery functions. See if you can figure it out. 

If there are other pressing features and you're having trouble with this, I can probably just implement this myself pretty quickly next week some time since this is just a usability improvement, not a bug fix or urgent feature.